cfnet/datasets.py

```diff
@@ -32,23 +31,63 @@ def _numpy_collate(batch):
   else:
     return np.array(batch)
 
-class NumpyLoader(DataLoader):
-  def __init__(self, dataset, batch_size=1,
-                shuffle=False, sampler=None,
-                batch_sampler=None, num_workers=0,
+class DataLoader:
+    def __init__(self,dataset, batch_size=1,
```

Comment:
Here, you should document each argument by writing a comment right after them.

For example:
```python
def __init__(
    self,
    dataset: Dataset, # dataset, a Dataset object
    batch_size: int=1, # batch size
    ...
)
```

Copy-paste all your comments right after each argument.

When running `nbdev_preview`, you will see the documents.